test/GoTrueClient.test.ts

```diff
@@ -1,4 +1,5 @@
 import { OpenIDConnectCredentials } from '../src'
+import { AuthError } from '../src/lib/errors';
```

Comment:
Errors are not exported by the library. Would it make sense to export them so a consumer can import errors and compare them with `instanceof AuthError` or `isAuthError(error)`?

Reply:
Good point – we definitely should at least be exporting  `isAuthError()`